docs/s3_test_results.md

```diff
@@ -336,32 +336,33 @@ Compatibility:  0/29/30
-| 10 | s3tests_boto3.functional.test_s3.test_multipart_upload_resend_part               | ERROR | ok     |
-| 11 | s3tests_boto3.functional.test_s3.test_multipart_upload_multiple_sizes            | ERROR | ok     |
-| 12 | s3tests_boto3.functional.test_s3.test_multipart_copy_multiple_sizes              | ERROR | ok     |
-| 13 | s3tests_boto3.functional.test_s3.test_multipart_upload_size_too_small            | ERROR | ok     |
-| 14 | s3tests_boto3.functional.test_s3.test_multipart_upload_contents                  | ERROR | ok     |
-| 15 | s3tests_boto3.functional.test_s3.test_multipart_upload_overwrite_existing_object | ERROR | ok     |
-| 16 | s3tests_boto3.functional.test_s3.test_abort_multipart_upload                     | ERROR | ok     |
-| 17 | s3tests_boto3.functional.test_s3.test_abort_multipart_upload_not_found           | ERROR | ok     |
-| 18 | s3tests_boto3.functional.test_s3.test_list_multipart_upload                      | ERROR | ok     |
-| 19 | s3tests_boto3.functional.test_s3.test_multipart_upload_missing_part              | ERROR | ok     |
-| 20 | s3tests_boto3.functional.test_s3.test_multipart_upload_incorrect_etag            | ERROR | ok     |
-| 21 | s3tests_boto3.functional.test_s3.test_multipart_resend_first_finishes_last       | ERROR | ERROR  |
-| 22 | s3tests_boto3.functional.test_s3.test_atomic_multipart_upload_write              | ERROR | ok     |
-
+| 9 | s3tests_boto3.functional.test_s3.test_multipart_upload_resend_part                | FAIL  | ok     |
+| 10 | s3tests_boto3.functional.test_s3.test_multipart_upload_multiple_sizes            | ok    | ok     |
+| 11 | s3tests_boto3.functional.test_s3.test_multipart_copy_multiple_sizes              | ok    | ok     |
+| 12 | s3tests_boto3.functional.test_s3.test_multipart_upload_size_too_small            | ok    | ok     |
+| 13 | s3tests_boto3.functional.test_s3.test_multipart_upload_contents                  | ok    | ok     |
+| 14 | s3tests_boto3.functional.test_s3.test_multipart_upload_overwrite_existing_object | ok    | ok     |
+| 15 | s3tests_boto3.functional.test_s3.test_abort_multipart_upload                     | ok    | ok     |
+| 16 | s3tests_boto3.functional.test_s3.test_abort_multipart_upload_not_found           | ok    | ok     |
+| 17 | s3tests_boto3.functional.test_s3.test_list_multipart_upload                      | ok    | ok     |
+| 18 | s3tests_boto3.functional.test_s3.test_multipart_upload_missing_part              | ok    | ok     |
+| 19 | s3tests_boto3.functional.test_s3.test_multipart_upload_incorrect_etag            | ok    | ok     |
+| 20 | s3tests_boto3.functional.test_s3.test_multipart_resend_first_finishes_last       | ERROR | ERROR  |
+| 21 | s3tests_boto3.functional.test_s3.test_atomic_multipart_upload_write              | ok    | ok     |
+| 22 | s3tests_boto3.functional.test_s3.test_multipart_copy_versioned                   | ok    | ok     |
+Comments: in [PR](https://github.com/nspcc-dev/s3-tests/pull/5)
 ## Tagging
```

Comment:
```suggestion
Comments: in [PR](https://github.com/nspcc-dev/s3-tests/pull/5)

## Tagging
```

Reply:
I mean new line between comments and tagging not between tests results and comments